package/Dockerfile

```diff
@@ -13,6 +13,9 @@ WORKDIR /var/lib/rancher
 
 ARG ARCH=amd64
 ARG IMAGE_REPO=rancher
+ARG SYSTEM_CHART_DEFAULT_BRANCH
```

Comment:
on master, this default should be `release-v2.2` for now and utlimately `release` when we have that branch
on release/v2.2 this should be set to `dev` so that RCs get dev by default